src/storage-mover/azext_storage_mover/aaz/latest/storage_mover/agent/_update.py

```diff
@@ -69,18 +67,12 @@ def _build_arguments_schema(cls, *args, **kwargs):
             options=["--description"],
             arg_group="Properties",
             help="A description for the Agent.",
-            nullable=True,
         )
         return cls._args_schema
 
     def _execute_operations(self):
         self.pre_operations()
-        self.AgentsGet(ctx=self.ctx)()
```

Comment:
Why switched from Generic update to Patch update?

Reply:
Swtiched back, should default to generic update, and only use patch if generic update not available.